src/Microsoft.ML.DataView/DataViewType.cs

```diff
@@ -119,6 +137,7 @@ public static NumberDataViewType SByte
         }
 
         private static volatile NumberDataViewType _instByte;
+        /// The singleton instance of the <see cref="NumberDataViewType"/> with representation type of <see cref="byte"/>.
         public static NumberDataViewType Byte
```

Comment:
You missed `<summary>` tags here and below.

Reply:
Oh boy. Thanks Eric. I should also probably add some comments about the singleton instances everywhere else...